fails:Kernel.loop is a private method
fails:Kernel.loop returns an enumerator if no block given
fails:Kernel.loop rescues StopIteration
fails:Kernel.loop rescues StopIteration's subclasses
